portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,43 @@
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+
+import org.eclipse.jetty.util.ajax.JSON;
+
 import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
+//   public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
```

Comment:
Don't leave old code commented like this. If you want to see what the code originally looked like, you can look through the history in Git.

Reply:
Remove old commented code.